samples/basic/blinky/src/main.c

```diff
@@ -10,7 +10,11 @@
 #include <gpio.h>
 
 /* Change this if you have an LED connected to a custom port */
+#ifdef LED0_GPIO_CONTROLLER
```

Comment:
Rather than supporting both, we should make LED0_GPIO_CONTROLLER the default, and have this be something like:

#ifndef LED0_GPIO_CONTROLLER
#define LED0_GPIO_CONTROLLER LED0_GPIO_PORT
#endif

#define LED_PORT LED0_GPIO_CONTROLLER